README.md

```diff
@@ -113,3 +113,15 @@ Run Cypress tests
 ```bash
 docker-compose run cypress
 ```
+
+## Run Percy Test
+
+To override the default baseline for certain builds , you will need to set PERCY_TARGET_BRANCH environment variable. The value of PERCY_TARGET_BRANCH will be the branch that has a Percy build which you want to use as a baseline.
+
+For example, if you want to compare your current checked out branch to the develop branch, you would set to PERCY_TARGET_BRANCH=develop. That will use the last Percy build on develop as the baseline to generate comparisons.
```

Comment:
we don't have develop branches only master

Reply:
we should probably create a .env file and add in there and add that to the repo. 

We should also add it to the instance that is running the percy tests with github actions 
Then it wouldn't need to be documented it would just work